.travis.yml

```diff
@@ -77,7 +77,7 @@ jobs:
 
     - name: Btcd Integration Windows
       script:
-        - make itest-parallel windows=1
+        - make itest-parallel windows=1 ITEST_PARALLELISM=2
```

Comment:
The change should be accompanied with a comment (or commit msg) with the rationale

Reply:
You're right, done.